build/task-build-modules.js

```diff
@@ -15,6 +16,11 @@ function copy(src) {
 export default async function() {
   logger.info(colors.cyan('Building node environment...'));
 
+  // Check if the `lib` directory exists. If it doesn't, need to rebuild.
+  if (!(await buildDirectoryExists())) {
```

Comment:
Should this be happening instead in Tasks.build?  Seems like the buildConfigHashes stuff is relevant to the other build tasks and not this one, so maybe it should happen here to be more clear: https://github.com/mozilla/tofino/blob/master/build/tasks.js#L41